macros/split_events.sql

```diff
@@ -98,7 +98,14 @@ where
     and {{ snowplow_utils.is_run_with_new_events("snowplow_web") }}
 {% endmacro %}
 
-{# {% macro databricks__split_events(event_name, flat_cols, sde_col, sde_keys, sde_types, context_cols, context_keys, context_types, context_aliases) %}
+{% macro databricks__split_events(event_name, flat_cols, sde_col, sde_keys, sde_types, context_cols, context_keys, context_types, context_aliases) %}
+
+{# Remove down to major version for Snowflake columns, drop 2 last _X values #}
```

Comment:
I guess they're not Snowflake columns but the same principle applies here, might be worth updating the comment